src/Toast.php

```diff
@@ -128,4 +128,17 @@ public function message($message, $level = null, $title = null)
         return $this;
     }
 
+    /**
+     * Clear toast messages arrays.
+     *
+     * @return $this
+     */
+    public function clear()
+    {
+        $this->toasts = [];
```

Comment:
This method should also include `session()->flash('toasts', $this->toasts);` to update the toasts stored in the session cache.

As it currently stands, it will only clear the internal array, but pending toasts are still stored in the session cache. If this method is to be useful outside of unit tests, it needs to flash the session cache too.